.clang-format

```diff
@@ -18,6 +18,7 @@ AlignOperands: false
 AllowShortFunctionsOnASingleLine: None
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
+AlwaysBreakBeforeMultilineStrings: true
```

Comment:
Why did you change this?

Reply:
@xerus2000 Please do NOT change those settings without any notice!!

Reply:
oh I'm sorry, I changed that for testing and forgot it

Reply:
Nonetheless, I would actually propose keeping it, it turns this:
![image](https://user-images.githubusercontent.com/13354331/82541361-a0760700-9b50-11ea-96d2-c05e6c590341.png)
into this:
![image](https://user-images.githubusercontent.com/13354331/82541382-aa980580-9b50-11ea-9baf-21072f37ffee.png)
which is much more readable imho

Reply:
I don't really care and think both are okay. 

@mixxxdj/developers opinions?

Reply:
@Holzhaus I reverted the change for now so we can merge :)

Reply:
Discussion moved: https://mixxx.zulipchat.com/#narrow/stream/109171-development/topic/About.20coding.20style/near/198313697